tests/foreman/ui/test_contenthost.py

```diff
@@ -256,22 +255,24 @@ def test_positive_end_to_end_bulk_update(session, vm):
             'description': description
         })
         session.hostcollection.associate_host(hc_name, vm.hostname)
+        # make a note of time for later CLI wait_for_tasks, and include
+        # 5 mins margin of safety.
+        timestamp = (datetime.utcnow() - timedelta(minutes=5)).strftime("%Y-%m-%d %H:%M")
         # Update the package by name
-        timestamp = datetime.utcnow()
         session.hostcollection.manage_packages(
             hc_name, content_type='Package',
             packages=FAKE_1_CUSTOM_PACKAGE_NAME,
             action='update_all'
         )
-        # Wait for upload profile event (in case system system slow)
+        # Wait for upload profile event (in case Satellite system slow)
         host = entities.Host().search(
             query={'search': 'name={}'.format(vm.hostname)})
         wait_for_tasks(
             search_query='label = Actions::Katello::Host::UploadPackageProfile'
                          ' and resource_id = {}'
                          ' and started_at >= "{}"'.format(
                              host[0].id, timestamp),
-            search_rate=10, max_tries=10,
+            search_rate=30, max_tries=10,
```

Comment:
I think `15-20` are good for search_rate rate and would help to save some time , what do you think ?

Reply:
OK, I will change to every 15 seconds.

 I checked with evgeni, the yum package_upload plug-in from the katello-hosts-tools.rpm does the upload and it just does it "post transaction", no specific time delay or polling interval. I thought there was a few mins delay.

Reply:
OK, done that. Bare in mind for a test that is going to pass, this only makes 15 seconds difference. For a test that is going to fail, its 2.5 mins difference.

Reply:
yeah it would save some seconds , as we have `max_tries` as well so we may utilize that if requires to increase , What do you think ?

Reply:
its OK, I made the change after consulting evgeni as to the expected profile upload time delay. 

Please merge and squash the commits. If it fails sometimes due to slow VMs or test system we can extend it.